proto/data_objects.proto

```diff
@@ -0,0 +1,268 @@
+
+syntax = "proto3";
+
+package ga4gh_data_object;
+
+// Import HTTP RESTful annotations
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+///-------------------------------------------------------------------
+message DataBundleObject {
+  //REQUIRED
+  string id = 1;
+
+  //REQUIRED
+  repeated DataObject files = 2;
+
+  //REQUIRED
+  // Timestamp of object creation in RFC3339.
+  google.protobuf.Timestamp created = 3;
+
+  // REQUIRED
+  // Timestamp of update in RFC3339, identical to create timestamp in systems
+  // that do not support updates.
+  google.protobuf.Timestamp updated = 4;
+
+  //R EQUIRED
+  // A string representing a version, some systems may use checksum, a RFC3339
+  // timestamp, or incrementing version number. For systems that do not support
+  // versioning please use your update timestamp as your version.
+  string version = 5;
```

Comment:
How exactly do we have a version? The what was the timestamp of the previous version? What was its hash value? Should we probably have a separate `DataObject repeated versions`  to hold different versions?

Reply:
map<string, string> system_metadata = N;
map<string, string> user_metadata = N;

Reply:
Added.